app/admin/education_experiences.rb

```diff
@@ -0,0 +1,41 @@
+ActiveAdmin.register EducationExperience do
+  permit_params :user_id, :institution, :course, :start_date, :end_date
+
+  filter :user
+  filter :course
+  filter :institution
+
+  menu parent: User.model_name.human(count: 2), priority: 2
+
+  index download_links: [:xlsx] do
+    column :user
+    column :institution
+    column :course
+    column :start_date
+    column :end_date
+
+    actions
+  end
+
+  form do |f|
+    user_id = params[:user_id] || f.object.user_id
```

Comment:
when is the `params[:user_id]` used?

Reply:
It's for adding new education experiences by the user experience tab.